app/controllers/reports/patient_lists_controller.rb

```diff
@@ -15,11 +15,11 @@ def show
     recipient_email = current_admin.email
     download_params = case region_class
     when "facility_group"
-      {id: @region.id}
+      {id: @region.source_id}
```

Comment:
Would this be a good time to stick to `@region.id`, and update the behavior of the `PatientListDownloadJob`?